analysis/R/read_input.R

```diff
@@ -101,8 +101,6 @@ ReadMapFile <- function(map_file, params = NULL, quote = "") {
   list(map = map, strs = strs, map_pos = map_pos)
 }
 
-# This function caches the .csv as an .rda for faster loading.  NOTE: It
```

Comment:
The comment is duplicated following the function's name. We use this style of commenting more often. I restored the "NOTE:..." comment.